Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Use a base image with platform specification
+FROM --platform=$BUILDPLATFORM debian:bookworm-slim
+
+# Define the arguments for Atmos version and platforms
+ARG TARGETPLATFORM
+ARG BUILDPLATFORM
+ARG ATMOS_VERSION
+
+SHELL ["/bin/bash", "-c"]
+# Update the package list and install curl and git
+RUN apt-get update && apt-get install -y curl git
```

Comment:
You might want to run hadolint on this dockerfile to fix small issues that will help reduce filesize and increase best practices

https://github.com/hadolint/hadolint

This can be added as a reviewdog action as well

https://github.com/reviewdog/action-hadolint

Reply:
Ok, will add this.

Reply:
Added + sarif